zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/mutable/MutableFormState.java

```diff
@@ -67,4 +74,11 @@ public interface MutableFormState extends FormState {
    * @param record the record of the form that is deleted
    */
   void deleteFormInFormKeyByFormIdAndDeploymentKeyColumnFamily(FormRecord record);
+
+  /**
+   * Deletes a form from FORM_KEY_BY_FORM_ID_AND_VERSION_TAG column family
+   *
+   * @param record the record of the form that is deleted
+   */
+  void deleteFormInFormKeyByFormIdAndVersionTagColumnFamily(FormRecord record);
```

Comment:
🔧 It would be nice if deletion-related code is in a separate commit for better PR readability (and future understanding of the changes).

Reply:
See https://github.com/camunda/camunda/pull/21494/commits/4a1c4f6bd7bb26a8c5e7b8d09dd5acca14c7ef8c.